src/main/java/com/example/usermanagement/Model/User.java

```diff
@@ -0,0 +1,105 @@
+package com.example.usermanagement.Model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+
+
+import java.time.LocalDateTime;
+
+@Data
```

Comment:
If you have added @Data then you don't require adding getter and setter functions separately. @Data is used to avoid boilerplate code.